generated/nitclk/nitclk/_converters.py

```diff
@@ -242,6 +243,53 @@ def convert_double_each_element(numbers):
     return [x * 2 for x in numbers]
 
 
+# buffer input to import buffer functions
+@singledispatch
+def _convert_import_buffer(value):  # noqa: F811
+    pass
+
+
+@_convert_import_buffer.register(list)  # noqa: F811
+@_convert_import_buffer.register(bytes)  # noqa: F811
+@_convert_import_buffer.register(bytearray)  # noqa: F811
+@_convert_import_buffer.register(array.array)  # noqa: F811
+def _(value):
+    return value
+
+
+@_convert_import_buffer.register(str)  # noqa: F811
+def _(value):
+    return value.encode()
+
+
+def convert_import_buffer(value):  # noqa: F811
+    import array
+    return array.array('b', _convert_import_buffer(value))
+
+
+# convert value to bytes
+@singledispatch
+def _convert_buffer_to_bytes(value):  # noqa: F811
+    pass
+
+
+@_convert_buffer_to_bytes.register(list)  # noqa: F811
```

Comment:
The names of the convert functions aren't symmetric.
`convert_to_foo` or `convert_foo_to_bar` seem to be the most used patterns.

Reply:
`convert_import_buffer` -> `convert_import_buffer_to_array`
`convert_buffer_to_bytes` => `convert_to_bytes`

First one is specific to `import_attributes_configuration_buffer()`, the second one is generic and can be used in multiple places.